schemas/session-replay/focus-record-schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "focus-record-schema.json",
+    "title": "FocusRecord",
+    "type": "object",
+    "description": "Schema of a Record type which contains focus information.",
+    "allOf": [
+        {
+            "$ref": "_common-record-schema.json"
+        },
+        {
+            "required": [
+                "type"
+            ],
+            "properties": {
+                "type": {
+                    "type": "integer",
+                    "description": "The type of this Record.",
+                    "const": 3,
+                    "readOnly": true
+                },
+                "data": {
+                    "type": "boolean",
+                    "description": "Whether this screen has a focus or not. For now it will always be true for mobile.",
+                    "const": true
+                }
```

Comment:
The web format uses `has_focus` attribute instead of `data: Bool`:
```typescript
data: {
   has_focus: boolean // always `true`
}
```
Do we want to change it?

Reply:
Nope, was just a mistake.